harvester_e2e_tests/fixtures/settings.py

```diff
@@ -15,7 +15,9 @@ def __init__(self):
 
         def _storage_net_configured(self):
             code, data = self.settings.get('storage-network')
-            if (cs := data.get('status', {}).get('conditions')):
```

Comment:
The assignment expressions (`:=`) is introduced in python 3.8, which is the version we targeting to support, is there any reason to replace it?

Reply:
It's not supported in Python 3.6. We maintain a Python 3.6 environment in our tox config. Either we drop that as well, or we stick to Python 3.6 compatibility.